### PR TITLE
Log Reader application

### PR DESCRIPTION
**Pull request by nmamro**

### CODE REVIEW

File: LogReader/LogReader/App.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public partial class App : Application
+    {
+        private readonly IHost host;
+
+        public App()
+        {
+            host = new HostBuilder()
+            .ConfigureServices((context, services) =>
+            {
+                services.AddScoped<LogViewModel>();
```
Comment (Cerberous007) [inline LogReader/LogReader/App.xaml.cs:20]: Does AddScoped work correctly?

File: LogReader/LogReader/Models/Log.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace LogReader.Models
```
Comment (Cerberous007) [inline LogReader/LogReader/Models/Log.cs]: Path is LogReader/LogReader?

File: LogReader/LogReader/Models/Log.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public class Log
```
Comment (Cerberous007) [inline LogReader/LogReader/Models/Log.cs]: Why do you have 2 identical models with different names (LogMessage and Log)?

File: Services/AuthenticationService/Commands/LogoutCommand.cs [hunk 4]
```diff
@@ -24,7 +24,7 @@ public bool Execute(Guid userId)
             if (userId == Guid.Empty)
             {
                 var e = new BadRequestException("Empty id in log out request.");
-                logger.LogWarning(e, "BadRequest thrown while trying to log out.");
+                logger.LogWarning(e, $"Attempt to logout user with empty id.");
```
Comment (Cerberous007) [inline Services/AuthenticationService/Commands/LogoutCommand.cs]: Why do you need $ here?

Comment (nmamro) [reply Services/AuthenticationService/Commands/LogoutCommand.cs]: > Why do you need $ here?

Don't need it here, thanks, will change in next commit

File: LogReader/LogReader/Models/Log.cs (hunk 2 above)
Comment (nmamro) [reply LogReader/LogReader/Models/Log.cs]: > Path is LogReader/LogReader?

I added a folder where I created the project, that's why there's such path

File: LogReader/LogReader/App.xaml.cs (hunk 1 above)
Comment (nmamro) [reply LogReader/LogReader/App.xaml.cs:20]: > Does AddScoped work correctly?

Why shouldn't it work?

File: LogReader/LogReader/Models/Log.cs (hunk 3 above)
Comment (nmamro) [reply LogReader/LogReader/Models/Log.cs]: > Why do you have 2 identical models with different names (LogMessage and Log)?

Log was automatically generated after database scaffolding, then didn't realize that it's better to use old LogMessage, changed it

File: .github/workflows/develop_build_actions.yml [hunk 5]
```diff
@@ -9,7 +9,7 @@ on:
 jobs:    
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```
Comment (ifelseelif) [inline .github/workflows/develop_build_actions.yml:12]: What it is?

Comment (nmamro) [reply .github/workflows/develop_build_actions.yml:12]: > What it is?

Workflow setting which enables wpf apps build, Ubuntu does not support building wpf so it was crashing

**Review by Cerberous007 — APPROVED** [review]

**Review by strannick-rodion — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

**Review by ifelseelif — APPROVED** [review]

**Pull request merged**